### ISSUE TITLE
Aggregate Hooks Issue

### ISSUE THREAD

**Issue by kingsleyzissou** [aerogear/offix#968]
## Bug Report
There are quite a few issues with the hooks. We have state in too many places and possibly the best solution is to strip back
the hooks, use the remote to fill the store:

- if we keep db as state provider
- remote will be refilling things
- and UI will just read state when it needs ( like react setState)

List of issues:
- Apollo Hooks vs Offix DataStore hooks #825
- Hooks causing some errors #798
- Using two the same hooks on the same page ends with render loop #875
- Lack of type definitions used in hooks despite using typescript files passed into hook #855

**Comment by wtrocki** [comment]
Yes. While this is 100% way to go we might actually really easily drop current implementation and get something from scratch. This is due to the fact that there might be some strange approaches in current hooks that might still leak out (useReducer etc.) so.. I would be easier to start from scratch.
